plot.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+from matplotlib.ticker import MultipleLocator
+
+data = np.loadtxt('stats', delimiter=',')
+years = data[:,0]
+sums = data[:,1]/1000
+diffs = data[:,2]/1000
+
+plt.style.use('ggplot')
```

Comment:
I think this should be split out to a function, so that we can do more plots in the future.

Reply:
OK, good idea. I will take care of that